cmd/scheduler/main_test.go

```diff
@@ -497,18 +497,18 @@ profiles:
 			},
 		},
 		{
-			name:            "single profile config - PodState - v1beta2",
+			name:            "single profile config - PodState - v1beta3",
 			flags:           []string{"--config", podStateConfigFile},
 			registryOptions: []app.Option{app.WithPlugin(podstate.Name, podstate.New)},
 			wantPlugins: map[string]*config.Plugins{
 				"default-scheduler": {
-					PreEnqueue: defaults.PluginsV1beta2.PreEnqueue,
-					QueueSort:  defaults.PluginsV1beta2.QueueSort,
-					Bind:       defaults.PluginsV1beta2.Bind,
-					PostFilter: defaults.PluginsV1beta2.PostFilter,
+					PreEnqueue: defaults.ExpandedPluginsV1beta3.PreEnqueue,
```

Comment:
why are we moving from `Plugins*` to `ExpandedPlugins*`? I see other occurrences in the old code, but still curious

Reply:
v1beta2 was removed in 1.28, so we have to turn to v1beta3. While in v1beta3, the variable `defaults.PluginsV1beta3` refers to a special plugin list with `MultiPoint` extension point:

```go
var PluginsV1beta3 = &config.Plugins{
	MultiPoint: config.PluginSet{...}
}
```

 so we have to use `ExpandedPluginsV1beta3` instead.

Reply:
thanks for the clarification